.pre-commit-config.yaml

```diff
@@ -15,6 +15,10 @@ repos:
     hooks:
         - id: ruff
           args: [ --fix ]
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v3.19.0
+    hooks:
+    -   id: pyupgrade
```

Comment:
this is superfluous as the pyupgrade ruleset in ruff is active see https://github.com/conda-forge/conda-smithy/blob/main/pyproject.toml#L50

Reply:
ruff also already defaults to Python 3.9 regarding formatting / fixes / upgrades as of >=0.8.0 https://astral.sh/blog/ruff-v0.8.0 in case not further specified.